core/cat/routes/memory.py

```diff
@@ -209,8 +218,8 @@ async def delete_memory_point(
 ) -> Dict:
     """Delete a specific point in memory"""
 
-    ccat = request.app.state.ccat
-    vector_memory = ccat.memory.vectors
+    vector_memory: VectorMemory = stray.memory.vectors
+    collections = list(vector_memory.collections.keys())
```

Comment:
the line `collections = list(vector_memory.collections.keys())` is duplicated of line 225.  Consider removing it